policy-controller.yaml

```
@@ -0,0 +1,47 @@
+package:
+  name: policy-controller
+  version: 0.6.2
+  epoch: 0
+  description: "The policy admission controller used to enforce policy on a cluster on verifiable supply-chain metadata from cosign."
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+      - make
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/sigstore/policy-controller/archive/v${{package.version}}/v${{package.version}}.tar.gz
```

```suggestion
      uri: https://github.com/sigstore/policy-controller/archive/refs/tags/v${{package.version}}.tar.gz
```

Tagged released archives are guaranteed not to change, changing the SHA, and `archive` links are not.

---

👍